tests/typecheck/topol/SmallArrayVP.java

```diff
@@ -0,0 +1,26 @@
+package typecheck.topol;
+
+import universe.qual.Any;
```

Comment:
What is this file testing? Are there some annotations and exptected errors missing?

Reply:
This was a smaller version of ArrayVP.java for my own understanding. I will delete this file.

Reply:
Did you decide against deleting this?

Reply:
So sorry, I was using a new command to copy changes across branches but the deletion didn't reflect properly.